common/azure-common-test/src/main/java/com/azure/common/test/models/RecordedData.java

```diff
@@ -88,4 +88,13 @@ public String removeVariable() {
             return variables.remove();
         }
     }
+
+    /**
+     * Getter method for {@link RecordedData#networkCallRecords}
+     *
+     * @return network call records in session record.
```

Comment:
I didn't add this method because its use is covered by the existing methods above such as findFirstAndRemoveNetworkCall and addRecord. Is there any reason to expose the underlying LinkedList?

Reply:
Deleted the one.

Reply:
I saw its usage in the old test cases and saw it as breaking the encapsulation OO design principle by unnecessarily exposing the underlying data type for ease of use.
https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)

Reply:
Good point. Removed it.